### ISSUE TITLE
Add option to remove entries when unfollowing a source

### ISSUE THREAD

**Issue by aaronpk** [indieweb/microsub#17]
When removing a source from a channel, Aperture provides an option in the UI to either remove all the existing entries or just stop adding new entries. It may be useful for this to be an option for Microsub clients as well.

![Aperture](https://media.aaronpk.com/Screen-Shot-2018-02-12-10-21-42-vGx0HU8iHg.jpg)

**Comment by EdwardHinkle** [comment]
Yeah, I probably wouldn’t put this incredibly high on the priority list, but I definitely think it should be in the spec for the long term.
